app/views/homepage/_government_activity.html.erb

```diff
@@ -51,6 +51,9 @@
             "track-dimension": "#{t("homepage.index.ministerial_departments_count")} #{t("homepage.index.ministerial_departments")}",
             "track-dimension-index": 29,
             "track-value": 1,
+            "index": 0,
+            "index_total": 2,
+            "section": t("homepage.index.departments_and_organisations")
```

Comment:
I'm confused here because looking at the code I don't think this should work, but looking at the deployment it appears to. I'd thought that the GA4 tracking attributes would need to be included in an object and then added here something like:

```
"ga4_link": { "attributes" }
```

Am I looking at the wrong thing? Or is it just Friday afternoon and I'm slipping?

Reply:
Yeah, this is incorrect but I think I addressed it in a later commit (apologies for the confusion). There is an instance in this template where I didn't address this as the component receiving the values handled building the `"ga4-link"` attribute object separately